tests/hopp/test_hydrogen/input_files/floris/floris_input_osw_18MW.yaml

```diff
@@ -0,0 +1,194 @@
+name: Gauss
+description: Three turbines using Gauss model
+floris_version: v3.0.0
+
+logging:
+  console:
+    enable: false
+    level: WARNING
+  file:
+    enable: false
+    level: WARNING
+
+solver:
+  type: turbine_grid
+  turbine_grid_points: 1
+
+farm:
+  layout_x:
```

Comment:
Is the layout being set by floris or by orbit? If it's being set by orbit we should remove this so there isn't confusion regarding the layout.

Reply:
The layout is still being set by ORBIT. I have removed the farm layout from the floors file to avoid confusion.